### PR TITLE
Add shared voicemail docs for Auto Attendant CallableEntity creation

### PR DESCRIPTION
**Pull request by amrElroumy**

### CODE REVIEW

File: skype/skype-ps/skype/New-CsAutoAttendantCallableEntity.md [hunk 1]
```diff
@@ -59,6 +60,14 @@ This example gets a user object using Get-CsOnlineUser cmdlet. We then use the A
 
 This example gets an application instance by name using Find-CsOnlineApplicationInstance cmdlet. We then use the AAD ObjectId of that application instance to create an application endpoint callable entity.
 
+### -------------------------- Example 5 --------------------------
+```powershell
+ $callableEntityId = (Find-CsGroup -SearchQuery "Main Auto Attendant") -MaxResults 1 -MailEnabledOnly $true | Select-Object -Property Id
+ $callableEntity = New-CsAutoAttendantCallableEntity -Identity $callableEntityId -Type SharedVoicemail -EnableTranscription
+```
+
+This example gets an mail-enabled group by name using Find-CsGroup cmdlet. We then use the ObjectId of that group to create a shared voicemail callable entity that supports transcription.
```
Comment (JensTrier) [inline skype/skype-ps/skype/New-CsAutoAttendantCallableEntity.md]: I think we need to be specific and mention that it needs to be O365 group for shared voicemail and not a normal group. Maybe even have an example finding the O365 group

Comment (amrElroumy) [reply skype/skype-ps/skype/New-CsAutoAttendantCallableEntity.md]: I'm not sure I understand the comment
> example finding the O365 group

isn't `$callableEntityId = (Find-CsGroup -SearchQuery "Main Auto Attendant") -MaxResults 1 -MailEnabledOnly $true | Select-Object -Property Id` sufficient?

Comment (JensTrier) [reply skype/skype-ps/skype/New-CsAutoAttendantCallableEntity.md]: There are different type of groups (security, distritbution, O365) you can create and find using Find-CsGroup. I just feel we need to specify that shared voicemail ONLY works with O365 groups

Comment (amrElroumy) [reply skype/skype-ps/skype/New-CsAutoAttendantCallableEntity.md]: How about after my latest commit?
I updated the docs to explicitly mention it has to be an O365 mail-enabled group.

`This example gets an O365 mail-enabled group by name using Find-CsGroup cmdlet. We then use the ObjectId of that group to create a shared voicemail callable entity that supports transcription.`

File: skype/skype-ps/skype/New-CsAutoAttendantCallableEntity.md [hunk 2]
```diff
@@ -59,6 +64,14 @@ This example gets a user object using Get-CsOnlineUser cmdlet. We then use the A
 
 This example gets an application instance by name using Find-CsOnlineApplicationInstance cmdlet. We then use the AAD ObjectId of that application instance to create an application endpoint callable entity.
 
+### -------------------------- Example 5 --------------------------
+```powershell
+ $callableEntityId = (Find-CsGroup -SearchQuery "Main Auto Attendant") -MaxResults 1 -MailEnabledOnly $true | Select-Object -Property Id
```
Comment (get-itips) [inline skype/skype-ps/skype/New-CsAutoAttendantCallableEntity.md]:
```suggestion
$callableEntityGroup=Find-CsGroup -SearchQuery "Main Auto Attendant" -ExactMatchOnly $true -MailEnabledOnly $true
```

File: skype/skype-ps/skype/New-CsAutoAttendantCallableEntity.md [hunk 3]
```diff
@@ -59,6 +64,14 @@ This example gets a user object using Get-CsOnlineUser cmdlet. We then use the A
...
+ $callableEntity = New-CsAutoAttendantCallableEntity -Identity $callableEntityId -Type SharedVoicemail -EnableTranscription
```
Comment (get-itips) [inline skype/skype-ps/skype/New-CsAutoAttendantCallableEntity.md]:
```suggestion
$callableEntity = New-CsAutoAttendantCallableEntity -Identity $CallableEntityGroup[0].Id -Type SharedVoicemail -EnableTranscription
```

File: skype/skype-ps/skype/New-CsAutoAttendantCallableEntity.md [hunk 4]
```diff
@@ -59,6 +64,14 @@ This example gets a user object using Get-CsOnlineUser cmdlet. We then use the A
...
+```
+
+This example gets an Office 365 group by name using Find-CsGroup cmdlet. We then use the ObjectId of that group to create a shared voicemail callable entity that supports transcription.
```
Comment (get-itips) [inline skype/skype-ps/skype/New-CsAutoAttendantCallableEntity.md]:
```suggestion
This example gets an Office 365 group by name using Find-CsGroup cmdlet. Then the Guid of that group is used to create a shared voicemail callable entity that supports transcription.
```

**Comment by kenwith** [comment]
@amrElroumy - thank you for adding this new content. Just a few things to update as suggested above. If you would please merge those into your PR I will get it merged.

**Pull request merged**